### DISCUSSION TITLE
Looking to contribute? Message me here.

### DISCUSSION THREAD

**Discussion by The-Best-Codes** [The-Best-Codes/best-bible#16] [General]
Hi guys! If you want to help develop `best-bible`, send a message here and explain why you'd like to help, your coding skills, and anything else you want to say. 

Then check out the [Contribution Guidelines](https://github.com/The-Best-Codes/best-bible/blob/main/CONTRIBUTING.md).

Thanks!

**Comment by rishitsheth1** [comment]
Hey does this have multiple types of books as well for translations, english, nkjv, kjv, etc. ?

**Reply by The-Best-Codes** [reply]
Hi @rishitsheth1, `best-bible` only supports KJV right now because the KJV is uncopyrighted. I can add other uncopyrighted versions (like the YLT, etc.) if you need them!
